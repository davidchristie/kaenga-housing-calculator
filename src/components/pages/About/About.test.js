import React from 'react'
import ReactTestUtils from 'react-addons-test-utils'

import About from './About'

const shallowRenderer = ReactTestUtils.createRenderer()

it('matches snapshot', () => {
  expect(shallowRenderer.render(<About />)).toMatchSnapshot()
})
