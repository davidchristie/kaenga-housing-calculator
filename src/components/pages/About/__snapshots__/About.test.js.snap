exports[`test matches snapshot 1`] = `
<Grid
  bsClass="container"
  className="About"
  componentClass="div"
  fluid={false}>
  <Col
    bsClass="col"
    componentClass="div"
    md={2} />
  <Col
    bsClass="col"
    componentClass="div"
    md={8}>
    <Jumbotron
      bsClass="jumbotron"
      className="About-header"
      componentClass="div">
      <h1>
        About
      </h1>
      <p>
        New Zealand housing is expensive. Part of the reason for this is that lots of people want to live here which drives up land prices. There is nothing we can do about that.
      </p>
      <p>
        But another part of the problem is that New Zealanders aren’t very innovative when it comes to housing. We’re still building homes based around the quarter-acre dream, despite the fact that people’s tastes are changing and these homes aren’t affordable.
      </p>
      <p>
        If you fill in this form, we’ll tell you more about your ideal home and provide suggestions for reducing your costs.
      </p>
      <p>
        You’ll get the most out of this form if you approach it with a firm budget in mind. That way you can adjust your home’s design to fit your budget. You can use 
        <a
          href="https://www.anz.com.au/personal/home-loans/calculators-tools/much-borrow/">
          ANZ’s calculator
        </a>
         to determine how much you might be able to borrow.
      </p>
    </Jumbotron>
  </Col>
  <Col
    bsClass="col"
    componentClass="div"
    md={2} />
</Grid>
`;
